### DISCUSSION TITLE
What is "_" in the translation?

### DISCUSSION THREAD

**Discussion by SnIPeRSnIPeR** [nicotine-plus/nicotine-plus#2188] [General]
Do we need this "_" in Russian translation. What is it?
![2022-09-16_15-11-18](https://user-images.githubusercontent.com/31162059/190590156-0dd716ed-4e8c-45ea-9851-6b2f9c54f51d.png)

**Comment by mathiascode** [comment]
https://nicotine-plus.org/doc/TRANSLATIONS.html#_underscores-in-strings
